controllers/setup.go

```diff
@@ -16,7 +16,10 @@ import (
 	"github.com/DataDog/datadog-operator/controllers/datadogagent"
 	componentagent "github.com/DataDog/datadog-operator/controllers/datadogagent/component/agent"
 	"github.com/DataDog/datadog-operator/pkg/config"
+
+	// NOTE: online clientpkg does not have what I need, may cause problems later
```

Comment:
what do you mean by this comment?

Reply:
I'm not quite sure. I went through the setup code and there was no difference between the setup of SLOs/Monitors and Dashboards. So I am assuming there is no issue, and I forgot to delete that.